thirdparty/KDDockWidgets/src/private/quick/TabBarQuick.cpp

```diff
@@ -124,7 +124,7 @@ QQuickItem *TabBarQuick::listView() const
 
     const QList<QQuickItem *> children = m_tabBarQmlItem->childItems();
     for (QQuickItem *child : children) {
-        if (qstrcmp(child->metaObject()->className(), "QQuickListView") == 0)
+        if (QString::fromUtf8(child->metaObject()->className()).startsWith(QString::fromUtf8("QQuickListView")))
```

Comment:
This is quite awful and hacky, but maybe the only way...

Slightly more beautiful would have been this: 
```c++
    const QMetaObject *listViewMetaObject = QMetaType::metaObjectForType(QMetaType::type("QQuickListView"));
    Q_ASSERT(listViewMetaObject);

    const QList<QQuickItem *> children = m_tabBarQmlItem->childItems();
    for (QQuickItem *child : children) {
        if (child->metaObject()->inherits(listViewMetaObject))
            return child;
    }
```
but that does not work (`listViewMetaObject` is nullptr).